### PR TITLE
Merge some job files and complete certain functionalities

### PR DESCRIPTION
**Pull request by DrMurx**
Picking up https://github.com/Map-A-Droid/MAD-ATV/pull/71

This PR groups some jobs into one file per feature, mainly those job which enable/disable a specific feature.

* Merge on/off/status jobs into one file per feature
* Add job to retrieve auto_update status
* Add jobs to modifiy Android settings, such as enable/disable HDMI control
* Simplify `ls -lL | awk {print $8}` into `ls -1L`

### CODE REVIEW

File: madatv_android_settings.json [hunk 1]
```diff
@@ -10,5 +10,17 @@
...
+    {
+      "TYPE": "jobType.PASSTHROUGH",
+      "SYNTAX": "su -c 'settings put global hdmi_control_enabled 1'"
+    }
+  ],
+  "ATV - HDMI control: disable": [
+    {
+      "TYPE": "jobType.PASSTHROUGH",
+      "SYNTAX": "su -c 'settings put global hdmi_control_enabled 0'"
+    }
```
Comment (krzee) [inline madatv_android_settings.json]: i dont think this is needed. 1.2.2 disabled hdmi-cec and there is no use case for needing it in scanners

Comment (DrMurx) [reply madatv_android_settings.json]: If we assume that everybody updates the ROM frequently, then you're right. I'll remove this section.

Comment (krzee) [reply madatv_android_settings.json]: we wont be putting a job for everything the rom updates. either they update or they can figure it out :shrug:

Comment (krzee) [reply madatv_android_settings.json]: the alternative would be to have a job for each thing done in 16mad, which is actually a valid thought... one could even make a single job that could do everything the mad rom does if they wanted. if that is the direction you want to go, then i guess i see no reason why not...

Comment (DrMurx) [reply madatv_android_settings.json]: Honestly I didn't really think that through :) Let's leave it out for now..

**Pull request merged**